examples/credential_metadata_sd_jwt_vc.json

```
@@ -0,0 +1,61 @@
+{
+    "credentials_supported": {
+        "SD_JWT_VC_example_in_OpenID4VCI": {
+            "format": "vc+sd-jwt",
+            "scope": "SD_JWT_VC_example_in_OpenID4VCI",
+            "cryptographic_binding_methods_supported": [
+                "did:example"
```

```suggestion
                "jwk"
```
best I can interpret 'cryptographic_binding_methods_supported' wrt https://www.ietf.org/archive/id/draft-ietf-oauth-sd-jwt-vc-01.html#section-3.2.2.2-3.5.1 `jwk` is the only value that'd be permitted in this context